Makefile

```diff
@@ -141,7 +141,9 @@ uninstall:
 	rm -rf $(DESTDIR)$(PREFIX)/lib/libmongoose.a $(DESTDIR)$(PREFIX)/lib/libmongoose.so.$(VERSION) $(DESTDIR)$(PREFIX)/include/mongoose.h $(DESTDIR)$(PREFIX)/lib/libmongoose.so
 
 mongoose.c: Makefile $(wildcard src/*) $(wildcard mip/*.c)
-	(cat src/license.h; echo; echo '#include "mongoose.h"' ; (for F in src/*.c mip/*.c ; do echo; echo '#ifdef MG_ENABLE_LINES'; echo "#line 1 \"$$F\""; echo '#endif'; cat $$F | sed -e 's,#include ".*,,'; done))> $@
+	(cat src/license.h; echo; echo '#include "mongoose.h"' ; \
+	echo; echo '#ifdef MG_ENABLE_LINES'; echo "#line 1 \"src/sock.c\""; echo '#endif'; cat src/sock.c | sed -e 's,#include ".*,,'; \
+	(for F in src/*.c mip/*.c ; do [ $$F != "src/sock.c" ] && (echo; echo '#ifdef MG_ENABLE_LINES'; echo "#line 1 \"$$F\""; echo '#endif'; cat $$F | sed -e 's,#include ".*,,'); done))> $@
```

Comment:
Please leave this intact. That does not worth it, to get a trivial S2PTR macro.

Reply:
INVALID_SOCKET is the one